pkg/koyeb/instances_exec.go

```diff
@@ -159,6 +169,42 @@ func (e *Executor) pushOne(ctx context.Context, c *websocket.Conn, r *ApiExecCom
 	return nil
 }
 
+func (e *Executor) pushTermResizes(ctx context.Context, c *websocket.Conn, from <-chan *TerminalSize) <-chan error {
+	errChan := make(chan error)
+	go func() {
+		if c == nil {
+			errChan <- errors.New("fatal: need an open connection")
+			return
+		}
+		for {
+			select {
+			case <-ctx.Done():
+				return
+			case r := <-from:
+				if r == nil {
+					errChan <- errors.New("cannot resize term to nil size")
+					return
+				}
+				resize := koyeb.NewExecCommandRequestTerminalSize()
+				resize.SetHeight(int32(r.Height))
```

Comment:
Maybe we could change the type of `TerminalSize` to use `int32` to avoid a double casting from `uint16` -> `int` -> `int32` ?

Reply:
Good point! Fixed